eclipse/sw360
# feat(ui): Generate clearing report button replacement

Signed-off-by: Smruti Sahoo <smruti.sahoo@siemens.com>


Ref: #911 
> * Did you add or update any new dependencies that are required for your change? - No dependencies added.

Issue: 

Fixes #911 

### How To Test?
 -> Please verify if generating product clearing report button is added in the "Obligations" tab as a standalone button.
-> Verify if the same is removed from the earlier place, (the navigation for which is mentioned in the issue)  .
->  Please do a sanity check on the generated report as well.

### Checklist
Must:
